### DISCUSSION TITLE
ArgoCD raises an error "Error: unknown flag: --parameter-file" when running "argocd app set my-app --parameter-file path/to/parameter-file.yaml"

### DISCUSSION THREAD

**Discussion by yuliya-roth** [argoproj/argo-cd#18670] [General]
Hey all,
I am using version:
argocd: v2.11.3+3f344d5
  BuildDate: 2024-06-06T12:31:55Z
  GitCommit: 3f344d54a4e0bbbb4313e1c19cfe1e544b162598

When I try to run  `argocd app set my-app --parameter-file path/to/parameter-file.yaml` 
ArgoCD throws this  "Error: unknown flag: --parameter-file" when running 
The same happens with `argocd app set my-app --from-file path/to/parameters.json`
Am I doing anything wrong?

Thanks for your help!

**Comment by crenshaw-dev** [comment]
Looks like documentation for those flags somehow got into the CLI even thought the flags don't exist. Fixing it here: https://github.com/argoproj/argo-cd/pull/18673

**Comment by dove-young** [comment]
I got same problem on version `v2.11.3+3f344d5`. So how can I set an app using YAML or Json file?
